frontend/src/components/ChatBot/ChunkInfo.tsx

```diff
@@ -10,9 +10,32 @@ import ReactMarkdown from 'react-markdown';
 import { generateYouTubeLink, getLogo, isAllowedHost } from '../../utils/Utils';
 import { ThemeWrapperContext } from '../../context/ThemeWrapper';
 import { chatModeLables } from '../../utils/Constants';
+import { useCredentials } from '../../context/UserCredentials';
+import GraphViewModal from '../Graph/GraphViewModal';
+import { handleGraphNodeClick } from './chatInfo';
+import { IconButtonWithToolTip } from '../UI/IconButtonToolTip';
 
 const ChunkInfo: FC<ChunkProps> = ({ loading, chunks, mode }) => {
   const themeUtils = useContext(ThemeWrapperContext);
+  const { userCredentials } = useCredentials();
+  const [neoNodes, setNeoNodes] = useState<any[]>([]);
+  const [neoRels, setNeoRels] = useState<any[]>([]);
+  const [openGraphView, setOpenGraphView] = useState(false);
+  const [viewPoint, setViewPoint] = useState('');
+  const [loadingGraphView, setLoadingGraphView] = useState(false);
```

Comment:
@prakriti-solankey here we are not using the state

Reply:
Resolved